api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java

```diff
@@ -1,26 +1,30 @@
 package org.openmrs.module.fhir.api.diagnosticreport.handler;
 
+import org.openmrs.Obs;
 import org.openmrs.module.fhir.api.diagnosticreport.DiagnosticReportHandler;
-import org.openmrs.module.fhir.api.diagnosticreport.DiagnosticReportTemplate;
 
-import ca.uhn.fhir.model.dstu2.resource.DiagnosticReport;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 public class LaboratoryHandler extends AbstractHandler implements DiagnosticReportHandler {
 	
+	public LaboratoryHandler() {
+		super();
+	}
+
 	@Override
-	public DiagnosticReport generateFHIRDiagnosticReport(DiagnosticReportTemplate omrsDiagnosticReport) {
-		return null;
+	public Set<Obs> saveObs(List<Obs> result) {
+		return new HashSet<Obs>();
 	}
-	
+
 	@Override
-	public DiagnosticReportTemplate generateOpenMRSDiagnosticReport(DiagnosticReport fhirDiagnosticReport) {
-		return null;
```

Comment:
@surangak I'm sorry that I unable to include full set of methods here. Other than saving Obs, it can save `presentedForm` field also (which is a PDF in this case). Then following methods needs to implement here
* createPresentedForm
* getPresentedForm
* updatePresentedForm
* deletePresentedForm